language/English/strings.po

```diff
@@ -12792,7 +12792,12 @@ msgctxt "#25009"
 msgid "The menu of this Blu-ray is not supported"
 msgstr ""
 
-#empty strings from id 25010 to 29799
+#: xbmc/video/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#25010"
```

Comment:
ah, Confluence is using it - it's showing "chapter 2/10" in the dialogseekbar and VideoFullScreen. But I guess we could as well simply change 21396 to use %s and update Confluence accordingly. I'm just not sure if other skins also rely on the label which could become an issue with backwards compatibility. Are I** skins still compatible with Helix? @ronie ?